### DISCUSSION TITLE
Load and Save Palettes…

### DISCUSSION THREAD

**Discussion by PerspexSphinx** [mriale/PyDPainter#180] [Ideas]
**Great Program**, I'm really enjoying it immensely.
​I was wondering if you are planning on adding the ability to load and save Palettes of various old and new formats?  
eg:
- .Pal Amiga (DPaint & Brilliance)
- .Col Amiga (Personal Paint)
- .Pal Modern (GrafX2, Aseprite) 
- .GPL (Gimp,  Inkscape)
- .Json (Pixelorama, Various)    

​Also the addition of a "Spline Tool" would be **Awesome!**
​

**Comment by mriale** [comment]
Great idea! Added issue #181 for loading/saving of color palettes. I'll have to do some research to find out the exact file layouts.

Also, the spline tool is still on the list at #85. Hmmm, that would be nice if I could use spline paths to animate brushes around the screen. Added that comment to #85.

Thanks for your ideas!
